doc/tools/west/west-apis.rst

```diff
@@ -5,6 +5,4 @@
 West APIs
 #########
 
-.. automodule:: west.runners.core
```

Comment:
No documentation in this file now?

Reply:
Fixed now; had to do some spelunking into the latest CI system